server/protocols/jmap/src/test/java/org/apache/james/jmap/utils/JsoupHtmlTextExtractorTest.java

```diff
@@ -53,14 +52,14 @@ public void toPlainTextShouldRemoveSimpleHtmlTag() {
     @Test
     public void toPlainTextShouldReplaceSkipLine() {
         String html = "<p>This is an<br/>HTML text !</p>";
-        String expectedPlainText = "This is an\nHTML text !\n";
+        String expectedPlainText = "This is an HTML text !";
```

Comment:
I'm pretty sure we use it in an other case. When there is a mail with HTML only, the text content is generated from HTML with the HtmlTextExtractor. And then it can be displayed by Inbox on mobiles for example. Please check.